packages/starlight/__e2e__/fixtures/git/astro.config.mjs

```diff
@@ -0,0 +1,11 @@
+import starlight from '@astrojs/starlight';
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+	integrations: [
+		starlight({
+			title: 'Basics',
```

Comment:
We usually name our fixtures based on what they test/setup, we could just call it Git here or something related to last updated to make it different from the Basics fixture.